### PR TITLE
rest key, v, version two

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

rest key as v

also adds L = LOOC to the hotkey info menu

## Why It's Good For The Game

I was asked to, plus moving your mouse across the screen is hard

## Changelog
:cl:
add: rest hotkey
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
Please god tell me there is a way to hotkey combat mode help intent right click as well.

**Comment by ghost** [comment]
C toggles combat mode
1 help intent
right click right click

File: interface/interface.dm [hunk 1]
```diff
@@ -124,7 +124,9 @@ Hotkey-Mode: (hotkey-mode must be on)
 \tm = me
 \tt = say
 \to = OOC
+\tl = LOOC
```
Comment (Linzolle) [inline interface/interface.dm]: uhh?

Comment (Seris02) [reply interface/interface.dm]: that's from another pr I made that got merged that I forgot to put in there, sorry, I should include that in the description

Comment (Linzolle) [reply interface/interface.dm]: L ISNT the looc hotkey though. CTRL+O is.

Comment (Linzolle) [reply interface/interface.dm]: nvm yes it is but only outside of hotkey mode. It's still CTRL+O in hotkey mode.

Comment (Linzolle) [reply interface/interface.dm]: actually, is the old_hotkeys mode even used at all?

Comment (Seris02) [reply interface/interface.dm]: I have
no fucking clue
but yes this should be in here

Comment (Ghommie) [reply interface/interface.dm]: hotkeys mode must be off for this keybinding to work. It doesn't belong to this catalog.

File: interface/interface.dm [hunk 2]
```diff
@@ -188,8 +190,10 @@ Hotkey-Mode: (hotkey-mode must be on)
...
```
Comment (Ghommie) [inline interface/interface.dm]: Same.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**